### DISCUSSION TITLE
Update feeds via php script

### DISCUSSION THREAD

**Discussion by Mikey8906** [FreshRSS/FreshRSS#5892] [General]
Hello
I did a clean install of FreshRSS. Empty folder on the server and clean DB. I was on 1.19 and bought - maybe a good time to upgrade.

I noticed my feeds don't refresh so i looked at the manual
I then ran actualize_script.php by hand on the console.

Result on the log file

PHP Fatal error:  Uncaught TypeError: ob_implicit_flush() expects parameter 1 to be int, bool given in /web/test2/app/actualize_script.php:7
Stack trace:
#0 /web/test2/app/actualize_script.php(7): ob_implicit_flush()
#1 {main}
  thrown in /web/test2/app/actualize_script.php on line 7

I didn't touch the update script. And it worked like a charm with 1.19. 
What am I missing?

**Comment by Alkarex** [comment]
Hello,
Please provide some information about your setup. OS, install type, versions...

**Comment by Alkarex** [comment]
Right, there has been a breaking change in PHP 8.0 https://php.net/ob-implicit-flush
You are probably running PHP 7.x, aren't you?

**Reply by Alkarex** [reply]
Could you try this patch? https://github.com/FreshRSS/FreshRSS/pull/5893

**Reply by Alkarex** [reply]
Merged in our edge branch. Tests & feedback welcome

**Reply by Mikey8906** [reply]
Yes I had PHP7. I simply took the step and upgraded to 8 and it all works. 
Sooner or later I had to do this anyway.
Seems to work again. I got my feeds back :)
